r/R/util.R

```diff
@@ -22,13 +22,6 @@ if (!exists("deparse1")) {
   }
```

Comment:
(Can't comment above here but) should we also drop this backport if we're bumping to support only R >= 4.0? It's also only used in the quantile binding plus some partitioning error messages.

Reply:
Aaah yes, thanks I didn't find this one when I did my search before.